datacapture/src/main/java/com/google/android/fhir/datacapture/fhirpath/FhirPathUtil.kt

```diff
@@ -47,11 +48,15 @@ internal fun evaluateToDisplay(expressions: List<String>, data: Resource) =
 internal fun evaluateToBoolean(
   questionnaireResponse: QuestionnaireResponse,
   questionnaireResponseItemComponent: QuestionnaireResponseItemComponent,
-  expression: String
-) =
-  fhirPathEngine.evaluateToBoolean(
-    questionnaireResponse,
-    null,
-    questionnaireResponseItemComponent,
-    expression
+  expression: String,
+  contextMap: Map<String, Base?> = mapOf(),
+): Boolean {
+  val expressionNode = fhirPathEngine.parse(expression)
+  return fhirPathEngine.evaluateToBoolean(
+    /* <appInfo> */ contextMap,
+    /* <focusResource> */ questionnaireResponse,
+    /* <rootResource> */ null,
+    /* <base> */ questionnaireResponseItemComponent,
+    /* <node> */ expressionNode,
```

Comment:
sorry to be pedantic and for not catching this earlier - but in our codebase we use the following syntax normally:

```suggestion
    /* appInfo= */ contextMap,
    /* focusResource= */ questionnaireResponse,
    /* rootResource= */ null,
    /* base= */ questionnaireResponseItemComponent,
    /* node= */ expressionNode,
```